### ISSUE TITLE
Support logical if() function (Less.js 3.0)

### ISSUE THREAD

**Issue by toviet** [wikimedia/less.php#111]
Hi, 

Is there any plan to support logical function that was added to less.js from version 3.0?

https://lesscss.org/functions/#logical-functions-if

Right now, when my less has an if condition i.e.:

```
if((@panelAlignment = flex-start), @exitPosition, auto);
```

I would get `Unexpected Input` exception when trying to parse the less content.

Thanks!

**Comment by Krinkle** [comment]
Yes, it's on our roadmap for the 5.x release in our issue tracker at https://phabricator.wikimedia.org/T288498.

Would you be interested in helping start the relevant portion from the test/assets/lessjs-3.13.1.js file, or https://github.com/less/less.js/pull/3079?

**Comment by toviet** [comment]
Thanks for your reply and it's great to know that a new release is coming to address this. 

Unfortunately I'm all tied up at the moment so not sure if I could join the testing, but will let you know if I'm available.

Regarding the timeframe, do you have any estimated date when the new release is going to be ready? Sorry for being curious.

**Comment by Krinkle** [comment]
@toviet Less.php 5.1.0 should be out within 3-4 weeks, which will release several more Less.js 3.x features.

**Comment by Krinkle** [comment]
@toviet This is now implemented thanks to @anny21, and released in Less.php 5.4.0.
